docs/azure_jumpstart_hcibox/AKS/_index.md

```diff
@@ -10,41 +10,64 @@ Azure Stack HCI can provide host infrastructure for [Azure Kubernetes Service cl
 
-  ![Screenshot showing Kubernetes token page in Azure portal](./enter_token_portal.png)
+- From the _HCIBox-Client_ virtual machine, open File Explorer and navigate to the "C:\HCIBox" folder. Right-click on "Configure-AksWorkloadCluster.ps1" and then select Open with Code
 
-  ![Screenshot showing Kubernetes resources in Azure portal](./k8s_resources_portal.png)
+  ![Screenshot showing edit with VSCode](./open_with_code.png)
 
-- We can deploy configurations for this workload cluster automatically from source using [GitOps](https://learn.microsoft.com/azure/azure-arc/kubernetes/tutorial-use-gitops-connected-cluster). HCIBox includes a script that will automatically configure a simple "Hello Arc" application and ingress controller on the AKS cluster.
+- Uncomment line 6 in the script and edit the placeholder value for the $aadgroupID parameter with the object ID of your Microsoft Entra group. Save the script and close VSCode when finished.
 
-- Open up Windows Explorer on the _HCIBox-Client_ VM and navigate to the _C:\HCIBox_ folder. From here, right click on the _Deploy-GitOps.ps1_ file and run with PowerShell.
+  ![Screenshot showing editing script](./edit_script.png)
 
-  ![Screenshot showing starting GitOps script](./deploy_gitops.png)
+- - From the _HCIBox-Client_ virtual machine, open File Explorer and navigate to the "C:\HCIBox" folder. Right-click on "Configure-AksWorkloadCluster.ps1" and then select "Run with PowerShell."
 
-  ![Screenshot showing running GitOps script](./deploying_gitops.png)
+  ![Screenshot showing clusters in resource group](./run_with_powershell.png)
 
-- Once this script completes you will have a "Hello-Arc" icon on desktop of AdminCenter computer. Open this link to see the Hello Arc application deployed on your cluster.
+- Let the script run its course. It will close automatically when complete.
 
-  ![Screenshot showing Hello Arc link](./hello_arc_desktop.png)
+  ![Screenshot showing script running](./run_configure_aks.png)
 
-  ![Screenshot showing Hello Arc app](./hello_arc.png)
+- Once complete, you should have an AKS workload cluster called _hcibox-aks_ in your HCIBox resource group.
+
+  ![Screenshot showing AKS in resource group](./aks_in_resource_group.png)
+
+- Click on the AKS cluster to view details such as Kubernetes version. "Status" may show connecting for some time while the cluster fully connects to Azure.
+
+  ![Screenshot showing cluster detail](./cluster_detail.png)
+
+- From Azure portal, open Cloud Shell and run the following command, using the name of your HCIBox resource group.
+
+  ```
+  az connectedk8s proxy -n hcibox-aks -g <name of your resource group>
+  ```
+
+  ![Screenshot showing run connectedk8s](./cloud_shell_proxy.png)
+
+- From Cloud Shell, click "New Session" and then in the new shell you will have kubectl access to your cluster. Try running some kubectl commands for yourself.
+
+  ![Screenshot showing kubectl proxy access](./cloud_shell_k8s.png)
```

Comment:
```suggestion
- Once complete, you should have an AKS workload cluster called _hcibox-aks_ in your HCIBox resource group.

  ![Screenshot showing AKS in resource group](./aks_in_resource_group.png)

- Click on the AKS cluster to view details such as the Kubernetes version. "Status" may show connecting for some time while the cluster fully connects to Azure.

  ![Screenshot showing cluster detail](./cluster_detail.png)

- From Azure portal, open Cloud Shell and run the following command, using the name of your HCIBox resource group.

  ```
  az connectedk8s proxy -n hcibox-aks -g <name of your resource group>
  ```

  ![Screenshot showing run connectedk8s](./cloud_shell_proxy.png)

- From Cloud Shell, click "New Session" and then you will have kubectl access to your cluster in the new shell. Try running some kubectl commands for yourself.

  ![Screenshot showing kubectl proxy access](./cloud_shell_k8s.png)
```

Reply:
Sorry but I am not following this comment.